lib/zipline/zip_generator.rb

```diff
@@ -42,10 +42,10 @@ def normalize(file)
         {url: file.url}
       elsif defined?(CarrierWave::SanitizedFile) && file.is_a?(CarrierWave::SanitizedFile)
         {file: File.open(file.path)}
-      elsif defined?(ActiveStorage::Blob) && file.is_a?(ActiveStorage::Blob)
-        {blob: file}
       elsif is_io?(file)
         {file: file}
+      elsif file.respond_to? :service_url
```

Comment:
can this go back to `elsif defined?(ActiveStorage::Blob) && file.is_a?(ActiveStorage::Blob)`